README.md

```diff
@@ -51,7 +51,11 @@ The following actuator endpoints are available:
 The application uses a Postgres 11 database which is managed by Flyway. The SpringBoot integration will automatically manage migrations, so we only need these commands for debugging or if the local database has become corrupted. 
 * *Clean schema* : `$ ./gradlew flywayClean`
 * *View details and status information about all migrations* : `$ ./gradlew flywayInfo`
-* *Fix `ERROR: function uuid_generate_v4() does not exist`*: Run `DROP EXTENSION "uuid-ossp";` in the database. This shouldn't happen given the `IF NOT EXISTS` in the offending migration but unfortunately there appears to be an issue with Postgres.
+
+### Known issues
+`ERROR: function uuid_generate_v4() does not exist`
+
+Run `DROP EXTENSION "uuid-ossp";` in the database. This shouldn't happen given the `IF NOT EXISTS` in the offending migration but unfortunately there appears to be an issue with Postgres. It seems to be that this issue only occurs when the migration is run against a Postgres instance against which the migration has already been run at some point in the past - even if the schema has been subsequently deleted or if it was created under a different schema name. This is why it only usually happens in transient environments where schemas are often set up and torn down without destroying the underlying database.
```

Comment:
This is unrelated to the main change but I wanted to add more detail around this issue given we learned a bit more about it the other day.